### PR TITLE
healthcheck benign error

### PR DESCRIPTION
**Pull request by baude**
clean up the reporting of a benign error that can occur when a container
is first stopped and then removed.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: libpod/healthcheck_linux.go [hunk 1]
```diff
@@ -61,7 +62,9 @@ func (c *Container) removeTimer() error {
 		return errors.Wrapf(err, "unable to get systemd connection to remove healthchecks")
 	}
 	defer conn.Close()
-	serviceFile := fmt.Sprintf("%s.timer", c.ID())
-	_, err = conn.StopUnit(serviceFile, "fail", nil)
+	timerFile := fmt.Sprintf("%s.timer", c.ID())
+	if _, err := conn.StopUnit(timerFile, "fail", nil); err != nil && strings.HasSuffix(err.Error(), ".timer not loaded.") {
+		return nil
+	}
 	return err
```
Comment (mheon) [inline libpod/healthcheck_linux.go]: This won't work - `err` is only defined in the scope of the if

Comment (mheon) [reply libpod/healthcheck_linux.go]: I think you need to do an 
```if err != nil {
  if strings.HasSuffix(err.Error, ...) 
  {
    ...
  } else { 
    return err
  } 
} 
return nil
```

Comment (baude) [reply libpod/healthcheck_linux.go]: right right

**Comment by jwhonce** [comment]
LGTM

**Comment by mheon** [comment]
Tests blowing up

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**